app/src/main/java/com/android/voyageur/ui/formFields/UserDropdown.kt

```diff
@@ -93,14 +131,71 @@ fun UserDropdown(
     DropdownMenu(
         expanded = expanded,
         onDismissRequest = { expanded = false },
-        modifier = Modifier.fillMaxWidth()) {
+        modifier = Modifier.fillMaxWidth().testTag("userDropDown")) {
           users.forEachIndexed { index, userPair ->
+            val user = userPair.first
+            val isAlreadyParticipant = selectedTrip?.participants?.contains(user.id) == true
```

Comment:
Consider adding an explicit null check here since selectedTrip can be null.

Reply:
`selectedTrip` shouldn't be null, since the `UserDropDown` can only be accessed in the `Settings` Screen, which the user can access only by selecting a trip in `Overview`. However, to prevent potential errors, I added an explicit null check (which should never happen) at the beginning of the composable, to prevent the app from crashing. Lmk if you think it's good like this.